File: app/src/main/java/cc/smartcash/wallet/Services/WebWalletAPIConfig.java
package cc.smartcash.wallet.Services;

import cc.smartcash.wallet.Utils.URLS;
import retrofit2.Retrofit;

public class WebWalletAPIConfig {

    private final Retrofit retrofit;

    public WebWalletAPIConfig() {
        this.retrofit = RetrofitConfig.getClient(URLS.URL_API_WEBWALLET);
    }

    public WebWalletAPIService getWebWalletAPIService() {
        return this.retrofit.create(WebWalletAPIService.class);
    }
}

File: app/src/main/java/cc/smartcash/wallet/Services/WebWalletAPIConfig.kt
package cc.smartcash.wallet.Services

import cc.smartcash.wallet.Utils.URLS
import retrofit2.Retrofit

class WebWalletAPIConfig {

    private val retrofit: Retrofit = RetrofitConfig.getClient(URLS.URL_API_WEBWALLET)

    val webWalletAPIService: WebWalletAPIService
        get() = this.retrofit.create(WebWalletAPIService::class.java)

}